### PR TITLE
Support OTP Geocoder

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
Makes use of https://github.com/opentripplanner/otp-ui/pull/535.

## Config updates (Breaking Change)
- `CUSTOM_PELIAS_URL` is removed, replaced with `TRANSIT_BASE_URL`
- new `TRANSIT_GEOCODER` either one of `pelias` or `otp`

### CODE REVIEW

File: handler.ts [hunk 1]
```diff
@@ -50,13 +51,19 @@ if (redis) redis.on('error', (err) => console.log('Redis Client Error', err))
...
 if (
-  typeof CUSTOM_PELIAS_URL !== 'string' ||
+  typeof TRANSIT_BASE_URL !== 'string' ||
   typeof GEOCODER_API_KEY !== 'string' ||
   typeof BUGSNAG_NOTIFIER_KEY !== 'string' ||
   typeof GEOCODER !== 'string'
 ) {
   throw new Error(
-    'Error: configuration variables not found! Ensure env.yml has been decrypted'
+    'Error: required configuration variables not found! Ensure env.yml has been decrypted'
```
Comment (binh-dam-ibigroup) [inline handler.ts:60]: Finish sentence with a period.

**Review by binh-dam-ibigroup — APPROVED** [review]

File: handler.ts [hunk 2]
```diff
@@ -29,15 +29,16 @@ const BugsnagPluginAwsLambda = require('@bugsnag/plugin-aws-lambda')
...
   GEOCODER,
   GEOCODER_API_KEY,
   REDIS_HOST,
   REDIS_KEY,
   SECONDARY_GEOCODE_EARTH_URL,
   SECONDARY_GEOCODER,
-  SECONDARY_GEOCODER_API_KEY
+  SECONDARY_GEOCODER_API_KEY,
+  TRANSIT_BASE_URL,
+  TRANSIT_GEOCODER
```
Comment (daniel-heppner-ibigroup) [inline handler.ts:41]: We really should be documenting these somewhere... "TRANSIT_GEOCODER" is not immediately very obvious to me.

**Review by daniel-heppner-ibigroup — APPROVED** [review]
Looks great to me, can you just update the README please?

**Comment by miles-grant-ibigroup** [comment]
Good call. Have updated the example config!

**Pull request merged**